.github/workflows/differential-shellcheck.yml

```diff
@@ -0,0 +1,26 @@
+# Doc: https://github.com/redhat-plumbers-in-action/differential-shellcheck#usage
+---
+
+name: Differential ShellCheck
+on:
+  pull_request:
+    branches: [main]
+
+permissions:
+  contents: read
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Repository checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Differential ShellCheck
+        uses: redhat-plumbers-in-action/differential-shellcheck@v3
+        with:
+          severity: warning
+          token: ${{ secrets.GITHUB_TOKEN }})
```

Comment:
```suggestion
name: "Differential ShellCheck"
on:
  pull_request:
    branches: [main]

permissions:
  contents: read

jobs:
  lint:
    name: "Differential Shell Check"
    runs-on: ubuntu-latest

    steps:
    - name: "Clone Repository"
      uses: actions/checkout@v3
      with:
        fetch-depth: 0

    - name: "Differential ShellCheck"
      uses: redhat-plumbers-in-action/differential-shellcheck@v3
      with:
        severity: warning
        token: ${{ secrets.GITHUB_TOKEN }})
```

Reply:
Ok, the github suggestion-feature is probably not as readable as I hope. Just minor formatting changes to adapt to our code-base. Notably: we do not indent yaml-arrays specially. And I would drop the header from the file, since the action actually shows the github-path to the redhat-plumbers-in-action repository, and I think it is ok to require people to follow that.

Reply:
Thank you, done.